core/src/main/java/org/apache/mahout/common/RandomWrapper.java

```diff
@@ -47,14 +65,21 @@ public void setSeed(long seed) {
     }
   }
 
+    /**
+     * Sets the seed to a standard value that can be used to ensure that tests are deterministic.
+     */
   void resetToTestSeed() {
     setSeed(STANDARD_SEED);
   }
 
+    /**
+     * @return the delegate random number generator.
+     */
   public RandomGenerator getRandomGenerator() {
     return random;
   }
 
+
   @Override
   protected int next(int bits) {
```

Comment:
While we're at it can we add a comment block to this method?

Reply:
Addressed.